.github/workflows/update_iree_requirement_pins.yml

```diff
@@ -22,6 +22,12 @@ jobs:
     runs-on: ubuntu-22.04
 
     steps:
+      - uses: actions/create-github-app-token@c1a285145b9d317df6ced56c09f525b5c2b6f755 # v1.11.1
+        id: generate-token
+        with:
+          app-id: ${{ secrets.CREATE_PULL_REQUEST_TOKEN_APP_ID }}
```

Comment:
BTW @marbre we discussed assigning a github team as a reviewer. For now I did _not_ grant this permission, but we can pretty easily add it if/when we want to do that.

> Under Organization permissions: Members select Access: Read-only.
> NOTE: Only needed if you would like add teams as reviewers to PRs.

https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens